ring-core/src/ring/middleware/cookies.clj

```diff
@@ -67,14 +67,59 @@
   [key value encoder]
   (encoder {key value}))
 
+(defprotocol ICookieMaxAge
+  (seconds [this]))
+
+(defprotocol ICookieExpireDate
+  (rfc822-format [this]))
+
+;; clj-time used to be a direct dependency of ring.
+;; This section is here to maintain backward compatibility.
```

Comment:
I think we can remove these comments. I don't think this needs an inline explanation, and further explanations can be handled by the commit message and associated PR.